src/Time/TimeInput.tsx

```diff
@@ -130,6 +130,7 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     textAlignVertical: 'center',
     width: 96,
+    padding:0
```

Comment:
I tried this code in `Android` and is doing anything (I don't know in `iOS`.
I think maybe is for your use case the problem